.travis.yml

```diff
@@ -2,6 +2,9 @@ language: node_js
 notifications:
   email: false
 sudo: false
+before_install:
+# We don't need to run Travis on files like the CHANGELOG
+- ./bin/check-skip-ci.sh
```

Comment:
I don't believe this will actually do anything here – this'll just exit like any other pre-install step (e.g. installing a dependency) and Travis will continue on to run the tests as usual.

I think this'll need to be part of the 'Run npm test' 'stage' – but it'll still need to exit cleanly to prevent Travis from marking the build as failed.

Happy to pair on this next week if that helps.

Reply:
Actually, it's more complicated than that, because I think `TRAVIS_COMMIT_RANGE` will always reflect the diff of the entire PR, not the diff since the last test was run. So this would help **PRs** that only change markdown files, but it won't prevent tests running for individual commits that only touch markdown files.

We also cancel pending builds when we push new commits to an existing branch, so I think it's probably safer just to always run the tests…